validation/src/test/scala/hmda/validation/engine/lar/quality/LarQualityEngineSpec.scala

```diff
@@ -25,4 +25,17 @@ class LarQualityEngineSpec
       }
     }
   }
+
+  property("There should be no duplicates in qualityEditsList") {
+    qualityEditsList.toSet.toList.length mustBe qualityEditsList.length
```

Comment:
`List` has a method called `distinct` that might be helpful here (instead of converting to a set, then back to a list)